Neos.Flow/Classes/ObjectManagement/Proxy/ProxyMethodGenerator.php

```diff
@@ -31,10 +32,13 @@ public static function fromReflection(\Laminas\Code\Reflection\MethodReflection
         return $instance;
     }
 
-    public static function copyMethodSignature(\Laminas\Code\Reflection\MethodReflection $reflectionMethod): static
+    public static function copyMethodSignatureAndDocblock(\Laminas\Code\Reflection\MethodReflection $reflectionMethod): static
```

Comment:
Was that method unused before this change? It has its name changed, but its use in this PR is "new". 🤔

Reply:
Indeed, some places (AOP) use the parent copyMethodSignature directly, this one was unused, I think it may have been in preparation of future changes but I guess we always want the docblock and if not can re-add a copy of it.